[BUG] Crash on Phone Boot

### Device Model

ALL

### OS Version

HyperOS 3.0+

### Android Version

Android 16

### App Version

Other

### Describe the Bug

The app is unable to initiate NotificationReaderService

### Steps to Reproduce

1.- Reboot Phone
2.- Crash report from system

### Crash Logs (Logcat)

```shell
SystemUptimeMs: 83010
Process: com.d4viddf.hyperbridge
PID: 5268
UID: 10342
Frozen: false
Flags: 0x20c8be44
Package: com.d4viddf.hyperbridge v17 (0.5.0-alpha03)
Foreground: No
Process-Runtime: 149
Timestamp: 2026-04-08 03:54:29.141+0200
Build: Xiaomi/goya_eea/goya:16/BP2A.250605.031.A3/OS3.0.10.0.WOEEUXM:user/release-keys
Crash-Handler: com.android.internal.os.RuntimeInit$KillApplicationHandler
Loading-Progress: 1.0
Dropped-Count: 0

java.lang.RuntimeException: Unable to create service com.d4viddf.hyperbridge.service.NotificationReaderService: java.lang.IllegalStateException: User 0 must be unlocked for widgets to be available
	at android.app.ActivityThread.handleCreateService(ActivityThread.java:5774)
	at android.app.ActivityThread.-$$Nest$mhandleCreateService(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2859)
	at android.os.Handler.dispatchMessage(Handler.java:110)
	at android.os.Looper.loopOnce(Looper.java:267)
	at android.os.Looper.loop(Looper.java:360)
	at android.app.ActivityThread.main(ActivityThread.java:10054)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:616)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1115)
Caused by: java.lang.IllegalStateException: User 0 must be unlocked for widgets to be available
	at android.os.Parcel.createExceptionOrNull(Parcel.java:3369)
	at android.os.Parcel.createException(Parcel.java:3345)
	at android.os.Parcel.readException(Parcel.java:3328)
	at android.os.Parcel.readException(Parcel.java:3270)
	at com.android.internal.appwidget.IAppWidgetService$Stub$Proxy.startListening(IAppWidgetService.java:795)
	at android.appwidget.AppWidgetHost.startListening(AppWidgetHost.java:251)
	at com.d4viddf.hyperbridge.data.widget.WidgetManager.init(WidgetManager.kt:37)
	at com.d4viddf.hyperbridge.service.NotificationReaderService.onCreate(NotificationReaderService.kt:117)
	at android.app.ActivityThread.handleCreateService(ActivityThread.java:5761)
	... 9 more
```

### Screenshots / Screen Recording

![Image](https://github.com/user-attachments/assets/45e6b344-63c6-4bdc-9663-46efc237f3c7)

### Checklist

- [x] I have checked that this issue has not already been reported.
- [x] I have granted all required permissions (Notification Access, Autostart).

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -51,6 +51,7 @@
             <intent-filter>
                 <action android:name="android.intent.action.BOOT_COMPLETED" />
                 <action android:name="android.intent.action.QUICKBOOT_POWERON" />
+                <action android:name="android.intent.action.LOCKED_BOOT_COMPLETED" />
             </intent-filter>
         </receiver>
 

--- a/app/src/main/java/com/d4viddf/hyperbridge/receiver/BootReceiver.kt
+++ b/app/src/main/java/com/d4viddf/hyperbridge/receiver/BootReceiver.kt
@@ -12,6 +12,7 @@ class BootReceiver : BroadcastReceiver() {
     override fun onReceive(context: Context, intent: Intent) {
         // Check for both standard boot and quick boot (some ROMs use quick)
         if (intent.action == Intent.ACTION_BOOT_COMPLETED ||
+            intent.action == Intent.ACTION_LOCKED_BOOT_COMPLETED ||
             intent.action == "android.intent.action.QUICKBOOT_POWERON") {
 
             Log.d("HyperBridge", "Boot completed detected.")

